test/central_planner_test.py

```diff
@@ -972,6 +973,262 @@ def test_priority_no_decrease_with_multiple_updates(self):
         self.sch.add_task(worker=WORKER, task_id='D', priority=6)
         self.check_task_order(['A', 'B', 'D', 'C'])
 
+    def task_id_str(self, family, args):
+        return task_id_str(family, args)
+
+    def test_aggregation(self, aggregate_type='csv', expected_args='1,3'):
+        self.sch.add_task(worker=WORKER, task_id='A(a=1)', family='A', params={'a': '1'}, batchable=True)
+        self.sch.add_task(worker=WORKER, task_id='A(a=3)', family='A', params={'a': '3'}, batchable=True)
+        self.sch.add_task_batcher(worker=WORKER, family='A', batcher_aggregate_args={'a': aggregate_type})
+        self.check_task_order([self.task_id_str('A', {'a': expected_args})])
+
+    def test_aggregation_min(self):
+        self.test_aggregation('min', '1')
+
+    def test_aggregation_max(self):
+        self.test_aggregation('max', '3')
+
+    def test_aggregation_range(self):
+        self.test_aggregation('range', '1-3')
```

Comment:
Updated to use an enum.

I also removed range, as I just included it because it was simple and seemed useful for date ranges but wouldn't actually be easy to use directly in that sort of case.